settings.py

```diff
@@ -0,0 +1,41 @@
+import logging
+#########################################
+#										#
+#										#
+#			Global Variables			#
+#										#
+#										#
+#########################################
+
+
+PREFIX = "::"
+TOKEN = "your_token"
+AUTHOR_ID=289426079544901633
+
+#can be both str and discord snowflake IDs.
+POLL_ALLOWED_CHANNELS = ["polls"]
+
+#emojis dict. May be possible to change incomprehensible unicode to other strings recognized by discord
+EMOJIS = {
+	"thumbsup": "\U0001f44d",
+	"thumbsdown": "\U0001f44e",
+	"shrug": "\U0001f937",
+}
+
```

Comment:
Maybe add a command to add or remove roles in `ADMIN_ROLE(S)`, `GESTION_ROLES`, `DEV_ROLES`

Reply:
See #4